lib/autodoc/document.rb

```diff
@@ -87,7 +87,7 @@ def request_header
 
     def request_header_from_http_prefix
       request.headers.inject({}) do |table, (key, value)|
-        if key.start_with?("HTTP_")
+        if key.to_s.start_with?("HTTP_")
           table.merge(key.gsub(/^HTTP_/, "") => value)
```

Comment:
`:HTTP_FOO` may cause NoMethodError (undefined method `gsub' for :HTTP_FOO:Symbol).

```suggestion
          table.merge(key.to_s.gsub(/^HTTP_/, "") => value)
```